tests/projects/spark-job/deployment/project.yml

```diff
@@ -1,11 +1,10 @@
 name: 'sparkJob'
 description: 'Job to enrich charge session with total amount earned'
 maintainer: ['MPyL']
-mpylVersion: 1.0.9
+mpylVersion: 1.4.0
 stages:
   build: Sbt Build
   test: Sbt Test
-  deploy: Kubernetes Spark Job Deploy
```

Comment:
uhm?

Reply:
I made it an override project. @nik-kashi's logic requires you to set the deployment stage in the overrides not in the main one

Reply:
How do you make something a 'overrride project'? I don't see any difference, guess you made it override compatible? Anyway, if we never need these projects to not be 'override projects' I guess it's fine

Reply:
I added these files: https://github.com/Vandebron/mpyl/pull/302/files#diff-cdf26c82e3b16f52bf356fb05bdae1771f035ffaba4de3a9b973e6e4dfb6c1ce next to the project.yml